docs/services/motion/_index.md

```diff
@@ -488,6 +427,164 @@ logger.Info("Orientation of myArm from the Motion Service:", myArmMotionPose.Pos
+- `destination` [(spatialmath.Pose)](https://pkg.go.dev/go.viam.com/rdk@v0.2.50/spatialmath#Pose): The destination, which can be any [Pose](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Pose) with respect to the SLAM map's origin.
+- `slamName` [(resource.Name)](https://pkg.go.dev/go.viam.com/rdk/resource#Name): The `"name"` of the [SLAM Service](/services/slam/) from which the SLAM map is requested.
+- `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(bool)](https://pkg.go.dev/builtin#bool): Whether the request to `MoveOnMap` was successful.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk@v0.2.50/services/motion#Service).
+
+```go {class="line-numbers linkable-line-numbers"}
+motionService, err := motion.FromRobot(robot, "builtin")
+
+// Define a destination Pose with respect to the origin of the map from the SLAM service "my_slam_service"
+myPose := spatialmath.NewPoseFromPoint(r3.Vector{Y: 10})
+
+// Move the base component "my_base" to the destination pose of Y=10, a location of (0, 10, 0) in respect to the origin of the map
+success, err := motionService.MoveOnMap(context.Background(), "my_base", myPose, "my_slam_service", nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### MoveOnGlobe
+
+Move a component to a destination GPS point, represented in geographic notation *(latitude, longitude)*.
+Use a [Movement Sensor](/components/movement-sensor/) to check the location of the robot.
+
+{{< alert title="Usage" color="tip" >}}
+
+`MoveOnGlobe()` is intended for use with the [Navigation Service](/services/navigation), providing autonomous GPS navigation for rover [bases](/components/base/).
+
+To use `MoveOnGlobe()`, your movement sensor must be able to measure the GPS location and orientation of the robot.
+
+Make sure the [movement sensor](/components/movement-sensor/) you use supports usage of the following methods in its {{< glossary_tooltip term_id="model" text="model's" >}} implementation of the [Movement Sensor API](/components/movement-sensor/#api).
+
+- It must support `GetPosition()` to report the robot's current GPS location.
+- It must **also** support **either** `GetCompassHeading()` or `GetOrientation()` to report which way the robot is facing.
+- If your movement sensor provides multiple methods, your robot will default to using the values returned by `GetCompassHeading()`.
+
+{{< /alert >}}
+
+{{< alert title="Stability Notice" color="alert" >}}
+
+This method is experimental.
+
+Specifying `obstacles` in a request to `MoveOnGlobe()` will cause an error if you configure a `"translation"` in the `"geometries"` of any of the `GeoObstacle` objects.
+The translation feature is not yet supported by the [Navigation Service](/services/navigation/).
```

Comment:
We do not plan to support translation, since you can already specify the location using the latitude and longitude.